charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -592,17 +592,82 @@ agent:
         "metrics_collected": {
           "kubernetes": {
             "enhanced_container_insights": true
-          },
-          "application_signals": { }
-        }
-      },
-      "traces": {
-        "traces_collected": {
-          "application_signals": { }
+          }
         }
       }
     }
 
+agentWorkloadsWindows:
+  - name: cloudwatch-agent-windows
+    nodeSelector:
+      kubernetes.io/os: windows
+    mode: daemonset
```

Comment:
`config:` is missing here. the daemonset template will use the default windows config if it's missing.